iaso/models/org_unit.py

```diff
@@ -482,6 +482,13 @@ def as_dict_for_csv(self):
             "org_unit_type": self.org_unit_type.name,
         }
 
+    def as_dict_for_completeness_stats_with_parent(self):
```

Comment:
Just a remark : Didn't we say we are gonna stop using the `to_dict` and `as_dict` methods and use serializers in general ?

Reply:
yes but... i used same pattern as Olivier, and the org unit ai is not using the serializer correctly i think